.gitlab-ci-darwin.yml

```diff
@@ -0,0 +1,95 @@
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
+  MODULE_MPI: "openmpi/p9/4.0.2-gcc_7.4.0"
+  NUM_MPI_PROC_TESTING: 2
+  OMP_NUM_THREADS: 40
+  CMAKE_BUILD_TYPE: "Release" 
+  Kokkos_ARCH_POWER9: "ON" 
+  Kokkos_ARCH_VOLTA70: "ON" 
+  Kokkos_ENABLE_CUDA: "ON" 
+  Kokkos_ENABLE_CUDA_UVM: "OFF"
+  Kokkos_ENABLE_CXX11: "ON" 
+  Kokkos_ENABLE_OPENMP: "ON" 
+  PARTHENON_DISABLE_HDF5: "OFF" 
+
+stages:
+  - performance-regression
+
+# Is performed before the scripts in the stages step
+before_script:
+  - >
+    env -i bash ./scripts/darwin/setup.sh
+    ${MODULE_COMPILER}
+    ${MODULE_MPI}
+  - >
+    env -i bash ./scripts/darwin/build_hdf5_parallel.sh
+    ${MODULE_COMPILER}
+    ${MODULE_MPI}
+
+.gcc-mpi-cuda-performance-regression:
+  variables: 
+    BUILD_DIR: "build_power9_perf_regression_gcc_mpi"
+    CMAKE_CXX_COMPILER: $CI_PROJECT_DIR/external/Kokkos/bin/nvcc_wrapper
+  script:
+    - echo "./scripts/darwin/build.sh" > build_cmd.txt
+    - echo "BUILD_DIR              ${BUILD_DIR}             " >> build_cmd.txt
+    - echo "CMAKE_BUILD_TYPE       ${CMAKE_BUILD_TYPE}      " >> build_cmd.txt
+    - echo "CMAKE_CXX_COMPILER     ${CMAKE_CXX_COMPILER}    " >> build_cmd.txt
+    - echo "Kokkos_ARCH_POWER9     ${Kokkos_ARCH_POWER9}    " >> build_cmd.txt
+    - echo "Kokkos_ARCH_VOLTA70    ${Kokkos_ARCH_VOLTA70}   " >> build_cmd.txt
+    - echo "Kokkos_ENABLE_CUDA     ${Kokkos_ENABLE_CUDA}    " >> build_cmd.txt
+    - echo "Kokkos_ENABLE_CUDA_UVM ${Kokkos_ENABLE_CUDA_UVM}" >> build_cmd.txt
+    - echo "Kokkos_ENABLE_CXX11    ${Kokkos_ENABLE_CXX11}   " >> build_cmd.txt
+    - echo "Kokkos_ENABLE_OPENMP   ${Kokkos_ENABLE_OPENMP}  " >> build_cmd.txt
+    - echo "NUM_MPI_PROC_TESTING   ${NUM_MPI_PROC_TESTING}  " >> build_cmd.txt
+    - echo "OMP_NUM_THREADS        ${OMP_NUM_THREADS}       " >> build_cmd.txt
+    - echo "PARTHENON_DISABLE_HDF5 ${PARTHENON_DISABLE_HDF5}" >> build_cmd.txt
+    - echo "MODULE_CMAKE           ${MODULE_CMAKE}          " >> build_cmd.txt 
+    - echo "MODULE_CLANG           ${MODULE_CLANG}          " >> build_cmd.txt
+    - echo "MODULE_COMPILER        ${MODULE_COMPILER}       " >> build_cmd.txt
+    - echo "MODULE_CUDA            ${MODULE_CUDA}           " >> build_cmd.txt
+    - echo "MODULE_MPI             ${MODULE_MPI}            " >> build_cmd.txt
+    - >
+      env -i bash ./scripts/darwin/build.sh
+      ${BUILD_DIR}
+      ${CMAKE_BUILD_TYPE}
+      ${CMAKE_CXX_COMPILER}
+      ${Kokkos_ARCH_POWER9}
+      ${Kokkos_ARCH_VOLTA70}
+      ${Kokkos_ENABLE_CUDA}
+      ${Kokkos_ENABLE_CUDA_UVM}
+      ${Kokkos_ENABLE_CXX11}
+      ${Kokkos_ENABLE_OPENMP}
+      ${NUM_MPI_PROC_TESTING}
+      ${OMP_NUM_THREADS}
+      ${PARTHENON_DISABLE_HDF5}
+      ${MODULE_CMAKE}
+      ${MODULE_CLANG}
+      ${MODULE_COMPILER}
+      ${MODULE_CUDA}
+      ${MODULE_MPI}
+  artifacts:
+    expire_in: 3 days
+    paths:
+      - ${CI_PROJECT_DIR}/${BUILD_DIR}/tst/regression/outputs/advection_convergence/advection-errors.dat
```

Comment:
Is this an old version of the file?
The performance regression results should be in ` .../tst/regression/outputs/advection_performance/performance.png` don't they?

Reply:
Ya, I need to correct this.